test/unit/integrations/moneybookers/moneybookers_notification_test.rb

```diff
@@ -59,8 +59,15 @@ def test_acknowledge
     assert notification.acknowledge
   end
 
+  def test_failure_reasons
+    Moneybookers::Notification::FAILED_REASON_CODES.each do |code, reason|
+      notification = Moneybookers::Notification.new(http_raw_data.sub(/failed_reason_code=0/, "failed_reason_code=#{code}"), :credential2 => 'secret')
+      assert_equal reason, notification.failure_reason
+    end
+  end
```

Comment:
please add a test specifically for a code that is not defined in the `FAILED_REASON_CODES`